nodejs-babel-loader/nodejs-babel-loader.spec

```diff
@@ -0,0 +1,84 @@
+%global npm_name babel-loader
+
+Name: nodejs-%{npm_name}
+Version: 6.2.4
+Release: 1%{?dist}
+Summary: babel module loader for webpack
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/babel/babel-loader
+Source0: http://registry.npmjs.org/babel-loader/-/babel-loader-6.2.4.tgz
+Source1: http://registry.npmjs.org/mkdirp/-/mkdirp-0.5.1.tgz
+Source2: http://registry.npmjs.org/object-assign/-/object-assign-4.1.0.tgz
+Source3: http://registry.npmjs.org/loader-utils/-/loader-utils-0.2.15.tgz
+Source4: http://registry.npmjs.org/json5/-/json5-0.5.0.tgz
+Source5: http://registry.npmjs.org/emojis-list/-/emojis-list-2.0.1.tgz
+Source6: http://registry.npmjs.org/minimist/-/minimist-0.0.8.tgz
+Source7: http://registry.npmjs.org/big.js/-/big.js-3.1.3.tgz
+Source8: babel-loader-6.2.4-registry.npmjs.org.tgz
+Requires: nodejs(engine)
+BuildRequires: nodejs-devel
+BuildRequires: nodejs-packaging
+BuildRequires: npm
+BuildRequires: npm(webpack)
+BuildRequires: npm(babel-core)
```

Comment:
I was under the wrong impression then, thanks. I thought BuildRequires for Foreman would pull their dependent BuildRequires just in case they are needed